cdap-hbase-spi/src/main/java/co/cask/cdap/spi/hbase/HBaseDDLExecutor.java

```diff
@@ -64,7 +70,10 @@ void createTableIfNotExists(TableDescriptor descriptor, @Nullable byte[][] split
     throws IOException;
 
   /**
-   * Enable the specified table if it is disabled.
+   * Enable the specified table if it is disabled. Changing the dataset specs results into
```

Comment:
This is called when an HBase backed dataset has its properties modified.
In order to modify the HBase table, CDAP first calls disableTableIfEnabled, then calls modifyTable, then enables the table with this method.